server/service/src/requisition/response_requisition/update.rs

```diff
@@ -47,9 +48,25 @@ pub fn update_response_requisition(
         .connection
         .transaction_sync(|connection| {
             let requisition_row = validate(connection, store_id, &input)?;
-            let updated_requisition = generate(user_id, requisition_row, input);
+            let updated_requisition = generate(user_id, requisition_row.clone(), input.clone());
             RequisitionRowRepository::new(&connection).upsert_one(&updated_requisition)?;
 
+            if input.status == Some(UpdateResponseRequstionStatus::Finalised)
```

Comment:
requisition_row.status != updated_requisition.status ?

Reply:
Thanks, i see the commit, but I think the whole condition can be `requisition_row.status != updated_requisition.status` no need to check input.status